### PR TITLE
Analytics doc gap fixes

### PR DESCRIPTION
**Pull request by addb**
* Added syntax for React, Cordova, Flutter code snippets.
* Renamed section names with API names and ordered it alphabetically for sorted table of contents on the right.
* Added param description for the APIs
* Cleanup

### CODE REVIEW

**Comment by kc2cheng** [comment]
Hey Arjun, I assume you will also change the order of the API so it is listed out in  alphabetical order under the table of content?

File: using-mobile-extensions/adobe-analytics/analytics-api-reference.md [hunk 1]
```diff
@@ -476,6 +609,18 @@ FlutterACPAnalytics.setVisitorIdentifier("yourVisitorId");
 
 ### setVisitorIdentifier
 
+**Syntax**
+
+```dart
```
Comment (nporter-adbe) [inline using-mobile-extensions/adobe-analytics/analytics-api-reference.md:614]: Should be `jsx` right?

File: using-mobile-extensions/mobile-core/mobile-core-api-reference.md [hunk 2]
```diff
@@ -245,10 +273,13 @@ Map<String, String> additionalContextData = new HashMap<String, String>();
 
 **Syntax**
 
-```objectivec
-+ (void) trackState: (nullable NSString*) state data: (nullable NSDictionary*) data;
+```swift
```
Comment (nporter-adbe) [inline using-mobile-extensions/mobile-core/mobile-core-api-reference.md:276]: Why `swift` instead of `objectivec` here?

**Comment by addb** [comment]
> Hey Arjun, I assume you will also change the order of the API so it is listed out in alphabetical order under the table of content?

Hi Stephen, yes good idea, will reorder it. Thanks!

**Pull request merged**